### DISCUSSION TITLE
Is 1.18.2 still going to be supported as long as it is in LTS

### DISCUSSION THREAD

**Discussion by Desoroxxx** [ldtteam/minecolonies#8887] [Q&A]
I play in 1.18.2 as some mods like to only port on LTS versions and will upgrade once they upgrade but are you going to backport changes made in 1.19 mine colonies to 1.18.2?

**Comment by Thodor12** [comment]
I have a strong feeling that will depend on Forge, they stated a little while back that 1.18.2 will remain LTS, but still allow work to be made on 1.19.2, so it's possilbe that when 1.20 comes around development will move to 1.20 in favor for 1.19.2, whilst keeping 1.18.2 as LTS.
But we don't know what exactly is going to happen, so it's too early to make conclusions on that yet.
